tracer/ebpf_integration_test.go

```diff
@@ -244,3 +245,11 @@ Loop:
 		})
 	}
 }
+
+func TestAllTracers(t *testing.T) {
+	coll, err := support.LoadCollectionSpec(false)
+	require.NoError(t, err)
+
+	_, _, err = initializeMapsAndPrograms(coll, tracertypes.AllTracers(), nil, false, 1, false, 0)
```

Comment:
This passes `nil` as kernelSymbols and that makes this fail on the kernels where they would be needed.

Reply:
already working on a fix - sorry that I didn't catch this earier.

Reply:
resolved with https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/205/commits/93960e56d48ca33dea857e41850a9b96bec8218e